compiler/rustc_trait_selection/src/traits/wf.rs

```diff
@@ -693,6 +693,10 @@ impl<'a, 'tcx> WfPredicates<'a, 'tcx> {
                         self.compute(ty.into());
                     }
                 }
+
+                ty::TyAlias(..) => {
+                    bug!("unexpected TyAlias in WfPredicates::compute");
+                }
```

Comment:
One of the most important parts of this PR is that WF handles `TyAlias` like it did before (i.e. check the WF of the RHS type), I don't understand how this handles that.
(If it's normalization, the `bug!` comment should say so)